wg-advisors/why/why_release_process.md

```diff
@@ -0,0 +1,30 @@
+# Release process
+
+## Summary 
+
+ASF project release software following a well defined process involving PMC members voting on release
+candidates, publishing the artifacts in the ASF distribution system, and announcing the release to the
+community.
+
+## Links to relevant documents
+
+* [Release Policy](https://www.apache.org/dev/release.html) 
+* [Release Distribution Policy](https://www.apache.org/dev/release-distribution.html)
+
+## Why are we doing it?
+
+We are doing it because this is what the ASF is doing. ASF delivers "Software for the public good"
+and the act of releasing software is a key part of that. The process is designed to ensure that the
+software is properly vetted and that the community is aware of the release and that the process of
+releasing software is transparent, open and secure. Releasing the software is a Legal Act of the 
+Foundation and it has consequences for the Foundation as software released by the Foundation is
+the main way the Foundation interacts with the public and can be hold accountable for the software
+it releases.
```

Comment:
I'd note it also gives the PMC and release manager some legal protection.

Reply:
Good point.